internal/testutil/base58.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Finobo
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+import (
+	"github.com/mailchain/mailchain/internal/encoding"
+)
+
+// MustDecodeBase58 decodes a Base58 string
+// It panics for invalid input.
+func MustDecodeBase58(input string) []byte {
+	dec, err := encoding.DecodeBase58(input)
+	if err != nil {
+		panic(err)
+	}
+
+	return dec
+}
```

Comment:
Hi Fred, the contents of this file are great but the location is causing a circular dependency. Can you move this file to `internal/encoding/encodingtest` and change `package testutil` to `package encodingtest`

Soon we will deprecate `testutil` in favor of `xxxxxtest` packages